src/components/victory-boxplot/helper-methods.js

```diff
@@ -367,7 +367,11 @@ const getBaseProps = (props, fallbackProps) => {
 
     TYPES.forEach((type) => {
       const labelText = getText(dataProps, type);
-      if (labelText !== null && typeof labelText !== undefined || !events || !sharedEvents) {
+      const labelProp = props.labels || props[`${type}Labels`];
+      if (
+        labelText !== null && typeof labelText !== undefined ||
```

Comment:
**BIG WARNING!!!**

I know this is existing code, but we have a very big problem here:

```js
> typeof labelText
< "undefined"
> typeof labelText !== undefined
< true
```

Can we create a high-priority ticket to scour **all** our victory repos for `== undefined` or something?

What we really want is:

```js
> typeof labelText !== "undefined"
< false
```

Reply:
Did a quick grep. I _think_ the only other place it occurs is in this same file in `master` at: https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-boxplot/helper-methods.js#L350

Side note -- we have lots of mixed `foo === undefined` and `typeof foo === "undefined"`. Might make sense to create a cross-repo ticket to hone all of them to one approach (prefer the latter or `_.isUndefined`).